confgenerator/fluentbit/inputs.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fluentbit
+
+import (
+	"fmt"
+	"path"
+	"strings"
+)
+
+// TODO: Move structs out of conf.go
+
+// DBPath returns the database path for the given log tag
+func DBPath(tag string) string {
+	// TODO: More sanitization?
+	dir := strings.ReplaceAll(strings.ReplaceAll(tag, ".", "_"), "/", "_")
+	return path.Join("${buffers_dir}", dir)
```

Comment:
This will have different results when run on Linux and Windows. The whole reason we introduced `filepathJoin` was to be able to run both Linux and Windows tests on either system without worrying about path separator discrepancies. In this case, the full semantics of `path.Join` don't apply anyway — this is just a string concatenation. Since we've determined that `/` works on Windows, can we just replace this with `return "${buffers_dir}/" + dir` instead?

Reply:
Never mind, I missed the fact that this was `path.Join` and not `filepath.Join`. Go is confusing.